modules/common-transformer-stream/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/stream/common/parquet/Codecs.scala

```diff
@@ -50,7 +50,7 @@ private[parquet] object Codecs {
       case FieldValue.DateValue(value) =>
         as[Date](value)
       case FieldValue.ArrayValue(values) =>
-        as[List[FieldValue]](values)
+        as[List[FieldValue]](values.toList)
```

Comment:
Can this be

```scala
as[Vector[FieldValue]](values)
```

to save from needlessly converting a vector to a list?  I haven't tried it.